proposals/0569-multiline-strings.rst

```diff
@@ -117,8 +121,6 @@ A working prototype is available at `brandonchinn178/string-syntax <https://gith
 
    #. Join the string back with ``\n`` delimiters
 
-      * Use ``\n`` regardless of the line terminators being used in the file. This matches the behavior of ``unlines``.
-
```

Comment:
I would prefer for the bullet to stay here, where it makes perfect sense. Perhaps replace "line terminators" by "lexical ``newline`` characters"